Jenkinsfile

```diff
@@ -20,27 +23,32 @@ pipeline {
         stage('Build') {
             steps {
                 configFileProvider([configFile(fileId: 'maven-settings-with-deploy-snapshot', variable: 'MAVEN_SETTINGS')]) {
-                    script {
-                        def mvnHome = tool 'Maven'
-                        sh "${mvnHome}/bin/mvn clean install -s $MAVEN_SETTINGS -Dmaven.test.failure.ignore=true || true"
-                        junit testDataPublishers: [[$class: 'ClaimTestDataPublisher']], testResults: '**/target/*-reports/*.xml'
-                        sh "${mvnHome}/bin/mvn clean -s $MAVEN_SETTINGS"
+                    timeout (time = 2, unit = 'HOURS') {
```

Comment:
Sure. I mainly wanted to experiment with block timeout in order to put a tighter limit on the stress tests, but I can't find a way to pass a timeout unit :)